README.md

```diff
@@ -79,11 +80,14 @@ This project supports below scenarios for end-to-end guest OS validation testing
 | Debian 9.x, 10.x                                |                                  |                          | :heavy_check_mark:                 |
 | Windows 10                                      | :heavy_check_mark:               |                          | :heavy_check_mark:                 |
```

Comment:
What about Windows 11?

Reply:
It's not GA yet, but since it's not confidential now I think it's ok to add it here.